server.py

```diff
@@ -1305,11 +1305,12 @@ async def relics_page2(ctx: ContextType, save: Savefile):
 async def seen_relic(ctx: ContextType, save: Savefile, *relic: str):
     """Output whether a given relic has been seen."""
     relic = " ".join(relic)
-
-    if sanitize(relic) in ["boat", "boatthingy", "boatthingie"]:
-        relics = ["Anchor", "Horn Cleat", "Captains Wheel"]
-    else:
-        relics = [relic]
+    relics = [relic]
+    
+    # Check if the relic is referencing a relic set:
+    relic_set = query(relic)
+    if relic_set.cls_name == RelicSet.cls_name:
```

Comment:
You could do a direct str comparison, i.e. `cls_name == "relic_set"`, or do `isinstance(relic_set, RelicSet)`. Latter more robust.